src/workos.ts

```diff
@@ -124,6 +128,13 @@ export class WorkOS {
                 error,
                 errorDescription,
               );
+            } else if (code && errors) {
+              throw new BadRequestException({
```

Comment:
I would much rather this be under `case 400: {`, but it breaks a unit test for `OauthException` ^.

We might be able to change it in the future, but we need to do some digging to ensure we don't break anything.

Reply:
Perhaps add a comment describing why we have this as a block instead of a `case 400:`